src/modules/Dashboard/features/Applet/Popups/AddParticipantPopup/AddParticipantPopup.tsx

```diff
@@ -179,9 +179,6 @@ export const AddParticipantPopup = ({
           title={t('addParticipant')}
           buttonText={t('next')}
           hasLeftBtn
```

Comment:
This should also be removed, else the button is still focusable and visible on hover:

```suggestion
```